### PR TITLE
Added .github folder with templates (#57)

### PR DESCRIPTION
**Pull request by grabear**
### Added the .github folder with appropriate files and templates:
* ISSUE_TEMPLATE.md
* CODE_OF_CONDUCT.md
* PULL_REQUEST_TEMPLATE.md
* CONTRIBUTING.md

Solves #32

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,3 +39,5 @@ install_github("vallenderlab/MicrobiomeR") # Install the package
 ## Notes
 
 * Code Style based on Google's R style developed by Hadley Wickham.
+* Please note that the [34m'MicrobiomeR'[39m project is released with a [Contributor Code of Conduct](.github/CODE_OF_CONDUCT.md). By contributing to this project, you agree to abide by its terms.
```
Comment (sdhutchins) [inline README.md]: Something formatting wise happened here 
```suggestion
* Please note that the `MicrobiomeR` project is released with a [Contributor Code of Conduct](.github/CODE_OF_CONDUCT.md). By contributing to this project, you agree to abide by its terms.
```

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Please briefly describe your problem and what output you expect.
+
+Please include a minimal reproducible example (AKA a reprex). If you've never heard of a [reprex](http://reprex.tidyverse.org/) before, start by reading <https://www.tidyverse.org/help/#reprex>.
+
+---
+
+Brief description of the problem
```
Comment (sdhutchins) [inline .github/ISSUE_TEMPLATE.md]:
```suggestion
## Brief description of the problem
```

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Please briefly describe your problem and what output you expect.
+
+Please include a minimal reproducible example (AKA a reprex). If you've never heard of a [reprex](http://reprex.tidyverse.org/) before, start by reading <https://www.tidyverse.org/help/#reprex>.
```
Comment (sdhutchins) [inline .github/ISSUE_TEMPLATE.md]:
```suggestion
Please include a minimal reproducible example (AKA a reprex - View [documentation](http://reprex.tidyverse.org/) about reprexes).
```

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+---
+
+Brief description of the problem
+
+```r
+# insert reprex here
+```
```
Comment (sdhutchins) [inline .github/ISSUE_TEMPLATE.md]: One quick idea after this is to add the below:

```markdown
## Session Information
<!-- Please provide session information in the code blocks below using the `sessionInfo()` command in your R console. -->
```
```
```console
# Insert Session info here
```
```

Comment (sdhutchins) [reply .github/ISSUE_TEMPLATE.md]: For whatever reason this won't show the closing 3 backticks...

Comment (grabear) [inline .github/ISSUE_TEMPLATE.md]:
```suggestion
```

## Session Information


```console
# Insert Session info here
```

```

**Pull request merged**